packages/@coorpacademy-components/src/molecule/video-player-mobile/index.native.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {StyleSheet, Text, View} from 'react-native';
+import JWPlayer from 'react-native-jw-media-player';
+
+import {Media} from '../questions/types';
+
+export type Props = {media: Media};
+
+const styleSheet = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'center',
+    marginTop: 20,
+    width: '100%'
+  },
+  video: {
+    flex: 1,
+    backgroundColor: '#f0f',
+    height: 500,
```

Comment:
the height should always be static ? 
( what happen for small phones ? )

Reply:
effectivement je vais vérifier 👍

Reply:
<p aligne="center">
<img width="247" alt="Screenshot 2023-02-06 at 16 37 31" src="https://user-images.githubusercontent.com/910636/217050798-130e3a62-0614-4d48-bf7b-154840ce37b0.png">

<img width="247" alt="Screenshot 2023-02-06 at 16 37 20" src="https://user-images.githubusercontent.com/910636/217050930-a4ab850f-3af7-4690-8fa8-7ada0e962cd7.png">
</p>

j'ai fait qques ajustements pour prendre un maximum de hauteur quand il y a la place, et pour avoir un minimum de hauteur quand les options sont plus grandes.